pandas_gbq/load.py

```diff
@@ -50,13 +50,16 @@ def encode_chunks(dataframe, chunksize=None):
 def load_chunks(
     client,
     dataframe,
+    project_id_table,
     dataset_id,
     table_id,
     chunksize=None,
     schema=None,
     location=None,
 ):
-    destination_table = client.dataset(dataset_id).table(table_id)
+    destination_table = (
+        client(project=project_id_table).dataset(dataset_id).table(table_id)
```

Comment:
Tests pass so presumably this must work — but are we sure `client` is callable here?

Reply:
You are totally right, `client` isn't callable here. When trying to use the code in a real example, I get the following error:
```
  File "to_gbq_simple.py", line 42, in <module>
    main(args.project_id, args.table_id)
  File "to_gbq_simple.py", line 33, in main
    pandas_gbq.to_gbq(df, table_id, project_id=project_id)
  File "/Users/vicente.reyes/hub/vreyespue/pandas-gbq/pandas_gbq/gbq.py", line 1222, in to_gbq
    progress_bar=progress_bar,
  File "/Users/vicente.reyes/hub/vreyespue/pandas-gbq/pandas_gbq/gbq.py", line 631, in load_data
    for remaining_rows in chunks:
  File "/Users/vicente.reyes/.pyenv/versions/3.7.5/lib/python3.7/site-packages/tqdm/std.py", line 1104, in __iter__
    for obj in iterable:
  File "/Users/vicente.reyes/hub/vreyespue/pandas-gbq/pandas_gbq/load.py", line 61, in load_chunks
    client(project=project_id_table).dataset(dataset_id).table(table_id)
TypeError: 'Client' object is not callable
```
I will try to find another way of implementing this.
Many thanks and best regards.

Reply:
Dear @max-sixty, I resolved the issue by using the following statement.
```
destination_table = client.dataset(dataset_id, project_id_table).table(
        table_id
    )
```
I also tested it in a real example, now it is working properly.
Many thanks and best regards.

Reply:
> Tests pass so presumably this must work — but are we sure client is callable here?

Unfortunately the secrets needed to run the system tests aren't available until merging, so many tests were skipped.